FIRA:
  Attack_Strategy: [0]
  Camera: {Focal: 11.144465290806757, High: 650.0}
  Center: {Camera_high: 650, Center_X: 369, Center_Y: 268, Front: 146, Inner: 35, Outer: 265,
    X: 332, Y: 258}
  Chase_Strategy: [0]
  Corner_Kick: [1.2, 60, 90, 50]
  Distance_Settings: [0, 0, 0]
  Dorsad_Attack: [8, 90, 180, 2, 1, 2]
  FPS: 25
  HSV:
    Ball: [0, 37, 28, 100, 20, 100]
    Blue: [205, 231, 80, 100, 42, 100]
    ColorMode: 1
    Green: [43, 187, 41, 74, 21, 100]
    White: [0, 359, 0, 359, 0, 359]
    Yellow: [10, 39, 0, 64, 3, 100]
    black: {angle: 50, gray: 50}
    white: {angle: 50, gray: 50}
  Parameterbutton: 1
  R1:
    Strategy: {tf_prefix: FIRA/R1, youbot_description: "<?xml version=\"1.0\" ?>\n\
        <!-- ===================================================================================\
        \ -->\n<!-- |    This document was autogenerated by xacro from /home/angus/FIRA16_clear/src/simulator/robot6th/robot6th_description/urdf/robot6th.xacro\
        \ | -->\n<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED          \
        \                       | -->\n<!-- ===================================================================================\
        \ -->\n<robot name=\"test_model\">\n  <!-- root link, on the ground just below\
        \ the model origin -->\n  <link name=\"base_footprint\">\n    <visual>\n \
        \     <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\n      <geometry>\n        <box\
        \ size=\"0.001 0.001 0.001\"/>\n      </geometry>\n    </visual>\n  </link>\n\
        \  <joint name=\"base_link_joint\" type=\"fixed\">\n    <origin rpy=\"0 0\
        \ 1.57\" xyz=\"0 0 1.25\"/>\n    <parent link=\"base_footprint\"/>\n    <child\
        \ link=\"base_link\"/>\n  </joint>\n  <!-- the model -->\n  <link name=\"\
        base_link\">\n    <inertial>\n      <mass value=\"50\"/>\n      <origin xyz=\"\
        0 0 -1.25\"/>\n      <inertia ixx=\"10\" ixy=\"0.0\" ixz=\"0.0\" iyy=\"10\"\
        \ iyz=\"0.0\" izz=\"100\"/>\n    </inertial>\n    <visual>\n      <origin\
        \ xyz=\"0 0 -1.73\"/>\n      <geometry>\n        <mesh filename=\"package://robot6th_description/meshes/robot6th.stl\"\
        \ scale=\"0.001 0.001 0.001\"/>\n      </geometry>\n    </visual>\n    <collision>\n\
        \      <origin xyz=\"0 0 -1.0\"/>\n      <geometry>\n        <cylinder length=\"\
        0.7\" radius=\"0.2\"/>\n      </geometry>\n    </collision>\n  </link>\n \
        \ <gazebo>\n    <plugin filename=\"libgazebo_ros_planar_move.so\" name=\"\
        object_controller\">\n      <commandTopic>PathPlan/RobotSpeed</commandTopic>\n\
        \      <odometryTopic>odom</odometryTopic>\n      <odometryFrame>odom</odometryFrame>\n\
        \      <odometryRate>20.0</odometryRate>\n      <robotBaseFrame>base_footprint</robotBaseFrame>\n\
        \    </plugin>\n  </gazebo>\n</robot>\n\n"}
  R2:
    Strategy: {tf_prefix: FIRA/R2, youbot_description: "<?xml version=\"1.0\" ?>\n\
        <!-- ===================================================================================\
        \ -->\n<!-- |    This document was autogenerated by xacro from /home/angus/FIRA16_clear/src/simulator/robot6th/robot6th_description/urdf/robot6th.xacro\
        \ | -->\n<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED          \
        \                       | -->\n<!-- ===================================================================================\
        \ -->\n<robot name=\"test_model\">\n  <!-- root link, on the ground just below\
        \ the model origin -->\n  <link name=\"base_footprint\">\n    <visual>\n \
        \     <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\n      <geometry>\n        <box\
        \ size=\"0.001 0.001 0.001\"/>\n      </geometry>\n    </visual>\n  </link>\n\
        \  <joint name=\"base_link_joint\" type=\"fixed\">\n    <origin rpy=\"0 0\
        \ 1.57\" xyz=\"0 0 1.25\"/>\n    <parent link=\"base_footprint\"/>\n    <child\
        \ link=\"base_link\"/>\n  </joint>\n  <!-- the model -->\n  <link name=\"\
        base_link\">\n    <inertial>\n      <mass value=\"50\"/>\n      <origin xyz=\"\
        0 0 -1.25\"/>\n      <inertia ixx=\"10\" ixy=\"0.0\" ixz=\"0.0\" iyy=\"10\"\
        \ iyz=\"0.0\" izz=\"100\"/>\n    </inertial>\n    <visual>\n      <origin\
        \ xyz=\"0 0 -1.73\"/>\n      <geometry>\n        <mesh filename=\"package://robot6th_description/meshes/robot6th.stl\"\
        \ scale=\"0.001 0.001 0.001\"/>\n      </geometry>\n    </visual>\n    <collision>\n\
        \      <origin xyz=\"0 0 -1.0\"/>\n      <geometry>\n        <cylinder length=\"\
        0.7\" radius=\"0.2\"/>\n      </geometry>\n    </collision>\n  </link>\n \
        \ <gazebo>\n    <plugin filename=\"libgazebo_ros_planar_move.so\" name=\"\
        object_controller\">\n      <commandTopic>PathPlan/RobotSpeed</commandTopic>\n\
        \      <odometryTopic>odom</odometryTopic>\n      <odometryFrame>odom</odometryFrame>\n\
        \      <odometryRate>20.0</odometryRate>\n      <robotBaseFrame>base_footprint</robotBaseFrame>\n\
        \    </plugin>\n  </gazebo>\n</robot>\n\n"}
  R3:
    Strategy: {tf_prefix: FIRA/R3, youbot_description: "<?xml version=\"1.0\" ?>\n\
        <!-- ===================================================================================\
        \ -->\n<!-- |    This document was autogenerated by xacro from /home/angus/FIRA16_clear/src/simulator/robot6th/robot6th_description/urdf/robot6th.xacro\
        \ | -->\n<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED          \
        \                       | -->\n<!-- ===================================================================================\
        \ -->\n<robot name=\"test_model\">\n  <!-- root link, on the ground just below\
        \ the model origin -->\n  <link name=\"base_footprint\">\n    <visual>\n \
        \     <origin rpy=\"0 0 0\" xyz=\"0 0 0\"/>\n      <geometry>\n        <box\
        \ size=\"0.001 0.001 0.001\"/>\n      </geometry>\n    </visual>\n  </link>\n\
        \  <joint name=\"base_link_joint\" type=\"fixed\">\n    <origin rpy=\"0 0\
        \ 1.57\" xyz=\"0 0 1.25\"/>\n    <parent link=\"base_footprint\"/>\n    <child\
        \ link=\"base_link\"/>\n  </joint>\n  <!-- the model -->\n  <link name=\"\
        base_link\">\n    <inertial>\n      <mass value=\"50\"/>\n      <origin xyz=\"\
        0 0 -1.25\"/>\n      <inertia ixx=\"10\" ixy=\"0.0\" ixz=\"0.0\" iyy=\"10\"\
        \ iyz=\"0.0\" izz=\"100\"/>\n    </inertial>\n    <visual>\n      <origin\
        \ xyz=\"0 0 -1.73\"/>\n      <geometry>\n        <mesh filename=\"package://robot6th_description/meshes/robot6th.stl\"\
        \ scale=\"0.001 0.001 0.001\"/>\n      </geometry>\n    </visual>\n    <collision>\n\
        \      <origin xyz=\"0 0 -1.0\"/>\n      <geometry>\n        <cylinder length=\"\
        0.7\" radius=\"0.2\"/>\n      </geometry>\n    </collision>\n  </link>\n \
        \ <gazebo>\n    <plugin filename=\"libgazebo_ros_planar_move.so\" name=\"\
        object_controller\">\n      <commandTopic>PathPlan/RobotSpeed</commandTopic>\n\
        \      <odometryTopic>odom</odometryTopic>\n      <odometryFrame>odom</odometryFrame>\n\
        \      <odometryRate>20.0</odometryRate>\n      <robotBaseFrame>base_footprint</robotBaseFrame>\n\
        \    </plugin>\n  </gazebo>\n</robot>\n\n"}
  RobotNumber: 1
  SCAN: {Angle_Near_Gap: 4, Angle_range_1: 1, Angle_range_2_3: 4, Dont_Search_Angle_1: 333,
    Dont_Search_Angle_2: 92, Dont_Search_Angle_3: 211, Magn_Far_End: 240, Magn_Far_Start: 165,
    Magn_Middle_Start: 117, Magn_Near_Gap: 1, Magn_Near_Start: 40}
  SPlanning_Velocity: [2.2, 0.3, 80, 50, 20, 3, 144, 5]
  Scan:
    Parameter: [4, 3, 40, 88, 142, 202, 87, 93, 207, 213, 327, 333]
  SideSpeedUp: [0.73, 1.2, 0.7, 5, 17.6]
  TypeS_Attack: [2, 0.2]
  TypeU_Attack: [0.8, 1, -90, 90, 0.6, 0.3, 10, 20]
  Zone_Attack: [1.5, 2.2]
  blackItem: {angle: 3, gray: 50}
FIRA_Behavior:
  Attack_Strategy: [30, 1, 1]
  Chase_Strategy: [3, 0.05, 0.08, 16.5, 0.35]
  Corner_Kick: [10, 0.45]
  Side_Speed_UP: [10, 0.45]
  TypeS_Attack: [20, 1, 1.5]
  TypeU_Chase: [30, 0.3]
  Zone_Attack: [0.6]
StrategySelection: [1, 0, 1, 0, 0, 0]
auto_adjust_stream_bytes_per_second: true
auto_exposure: false
auto_gain: false
auto_whitebalance: false
binning_x: 1
binning_y: 1
camera:
  image:
    compressed: {format: jpeg, jpeg_quality: 80, png_level: 9}
    compressedDepth: {depth_max: 10.0, depth_quantization: 100.0, png_level: 9}
    theora: {keyframe_frequency: 64, optimize_for: 1, quality: 31, target_bitrate: 800000}
  image_raw:
    compressed: {format: jpeg, jpeg_quality: 80, png_level: 9}
    compressedDepth: {depth_max: 10.0, depth_quantization: 100.0, png_level: 9}
    theora: {keyframe_frequency: 64, optimize_for: 1, quality: 31, target_bitrate: 800000}
exposure: 0.025
exposure_auto_max: 0.5
exposure_auto_target: 50
fixed_rate: 15.0
frame_id: ''
gain: 0
gain_auto_max: 19
gain_auto_target: 50
gazebo: {auto_disable_bodies: false, cfm: 0.0, contact_max_correcting_vel: 100.0,
  contact_surface_layer: 0.001, erp: 0.2, gravity_x: 0.0, gravity_y: 0.0, gravity_z: -9.8,
  max_contacts: 20, max_update_rate: 1000.0, sor_pgs_iters: 50, sor_pgs_precon_iters: 0,
  sor_pgs_rms_error_tol: 0.0, sor_pgs_w: 1.3, time_step: 0.001}
height: 493
prosilica_driver: {auto_adjust_stream_bytes_per_second: true, auto_exposure: false,
  auto_gain: false, auto_whitebalance: false, binning_x: 1, binning_y: 1, exposure: 0.025,
  exposure_auto_max: 0.5, exposure_auto_target: 50, fixed_rate: 15.0, frame_id: '',
  gain: 0, gain_auto_max: 19, gain_auto_target: 50, height: 493, stream_bytes_per_second: 45000000,
  trig_rate: 15.0, trig_timestamp_topic: '', trigger_mode: streaming, whitebalance_blue: 100,
  whitebalance_red: 100, width: 659, x_offset: 0, y_offset: 0}
rosbridge_websocket: {address: '', authenticate: false, delay_between_messages: 0,
  fragment_timeout: 600, max_message_size: None, port: 9090, retry_startup_delay: 5}
rosdistro: 'indigo

  '
roslaunch:
  uris: {host_angus_aspire_5742g__48051: 'http://angus-Aspire-5742G:48051/', host_angus_aspire_5742g__55935: 'http://angus-Aspire-5742G:55935/',
    host_iclab_desktop__33237: 'http://iclab-desktop:33237/', host_iclab_desktop__33506: 'http://iclab-desktop:33506/',
    host_iclab_desktop__33568: 'http://iclab-desktop:33568/', host_iclab_desktop__33695: 'http://iclab-desktop:33695/',
    host_iclab_desktop__33955: 'http://iclab-desktop:33955/', host_iclab_desktop__34557: 'http://iclab-desktop:34557/',
    host_iclab_desktop__36104: 'http://iclab-desktop:36104/', host_iclab_desktop__37341: 'http://iclab-desktop:37341/',
    host_iclab_desktop__37888: 'http://iclab-desktop:37888/', host_iclab_desktop__38032: 'http://iclab-desktop:38032/',
    host_iclab_desktop__38595: 'http://iclab-desktop:38595/', host_iclab_desktop__38844: 'http://iclab-desktop:38844/',
    host_iclab_desktop__38966: 'http://iclab-desktop:38966/', host_iclab_desktop__38987: 'http://iclab-desktop:38987/',
    host_iclab_desktop__39094: 'http://iclab-desktop:39094/', host_iclab_desktop__39606: 'http://iclab-desktop:39606/',
    host_iclab_desktop__40833: 'http://iclab-desktop:40833/', host_iclab_desktop__41464: 'http://iclab-desktop:41464/',
    host_iclab_desktop__41942: 'http://iclab-desktop:41942/', host_iclab_desktop__42001: 'http://iclab-desktop:42001/',
    host_iclab_desktop__42089: 'http://iclab-desktop:42089/', host_iclab_desktop__42535: 'http://iclab-desktop:42535/',
    host_iclab_desktop__44452: 'http://iclab-desktop:44452/', host_iclab_desktop__44621: 'http://iclab-desktop:44621/',
    host_iclab_desktop__44954: 'http://iclab-desktop:44954/', host_iclab_desktop__45644: 'http://iclab-desktop:45644/',
    host_iclab_desktop__45691: 'http://iclab-desktop:45691/', host_iclab_desktop__45996: 'http://iclab-desktop:45996/',
    host_iclab_desktop__46546: 'http://iclab-desktop:46546/'}
rosversion: '1.11.20

  '
run_id: 931121b6-3a33-11e7-83cf-b8aeed7cea2c
stream_bytes_per_second: 45000000
tf_prefix: FIRA/R3
trig_rate: 15.0
trig_timestamp_topic: ''
trigger_mode: streaming
use_sim_time: true
whitebalance_blue: 100
whitebalance_red: 100
width: 695
x_offset: 0
y_offset: 0
youbot_description: "<?xml version=\"1.0\" ?>\n<!-- ===================================================================================\
  \ -->\n<!-- |    This document was autogenerated by xacro from /home/angus/FIRA16_clear/src/simulator/robot6th/robot6th_description/urdf/robot6th.xacro\
  \ | -->\n<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                \
  \                 | -->\n<!-- ===================================================================================\
  \ -->\n<robot name=\"test_model\">\n  <!-- root link, on the ground just below the\
  \ model origin -->\n  <link name=\"base_footprint\">\n    <visual>\n      <origin\
  \ rpy=\"0 0 0\" xyz=\"0 0 0\"/>\n      <geometry>\n        <box size=\"0.001 0.001\
  \ 0.001\"/>\n      </geometry>\n    </visual>\n  </link>\n  <joint name=\"base_link_joint\"\
  \ type=\"fixed\">\n    <origin rpy=\"0 0 1.57\" xyz=\"0 0 1.25\"/>\n    <parent\
  \ link=\"base_footprint\"/>\n    <child link=\"base_link\"/>\n  </joint>\n  <!--\
  \ the model -->\n  <link name=\"base_link\">\n    <inertial>\n      <mass value=\"\
  50\"/>\n      <origin xyz=\"0 0 -1.25\"/>\n      <inertia ixx=\"10\" ixy=\"0.0\"\
  \ ixz=\"0.0\" iyy=\"10\" iyz=\"0.0\" izz=\"100\"/>\n    </inertial>\n    <visual>\n\
  \      <origin xyz=\"0 0 -1.73\"/>\n      <geometry>\n        <mesh filename=\"\
  package://robot6th_description/meshes/robot6th.stl\" scale=\"0.001 0.001 0.001\"\
  />\n      </geometry>\n    </visual>\n    <collision>\n      <origin xyz=\"0 0 -1.0\"\
  />\n      <geometry>\n        <cylinder length=\"0.7\" radius=\"0.2\"/>\n      </geometry>\n\
  \    </collision>\n  </link>\n  <gazebo>\n    <plugin filename=\"libgazebo_ros_planar_move.so\"\
  \ name=\"object_controller\">\n      <commandTopic>PathPlan/RobotSpeed</commandTopic>\n\
  \      <odometryTopic>odom</odometryTopic>\n      <odometryFrame>odom</odometryFrame>\n\
  \      <odometryRate>20.0</odometryRate>\n      <robotBaseFrame>base_footprint</robotBaseFrame>\n\
  \    </plugin>\n  </gazebo>\n</robot>\n\n"
